### ISSUE TITLE
RCS1229 is not reported while using the new using var syntax

### ISSUE THREAD

**Issue by cezarypiatek** [JosefPihrt/Roslynator#726]
**Product and Version Used**:  3.0.0

[RCS1229](https://github.com/JosefPihrt/Roslynator/blob/master/docs/analyzers/RCS1229.md) is not reported while using the new `using var` syntax:

```csharp
private async Task<int> DoSomething()
{
    using var service = CreateService();
    return service.GetAsync(); // Should report RCS1229 because of missing await keyword
}
```

It's working fine with old-style using block

```cs
private async Task<int> DoSomething()
{
    using (var service = CreateService())
    {
        return service.GetAsync(); // RCS1229  is reported as expected
    }
}
```
